index.js

```diff
@@ -0,0 +1,29 @@
+import showClock from './modules/date-time.js';
+import { renderBooks, addBook } from './modules/main.js';
+import {
+  addLink,
+  listLink,
+  contactLink,
+  addBtn,
+} from './modules/select-elements.js';
+import {
+  addToList,
+  displayBookList,
+  displayContactInfo,
+} from './modules/toggle-display.js';
+
+// Show clock
+showClock(); // Show clock initially
+setInterval(showClock, 1000); // Update the clock and HTML element's content every second
+
+// Display book list
+renderBooks();
+displayBookList();
+
+// Toggle display
+addLink.addEventListener('click', addToList);
+listLink.addEventListener('click', displayBookList);
+contactLink.addEventListener('click', displayContactInfo);
+
+// Add book
+addBtn.addEventListener('click', addBook);
```

Comment:
**[OPTIONAL BUT HIGHLY RECOMMENDED]**

- [ ] Great work on making the add and remove function work as expected 👍 but there is a minor issue here 😨 Whenever we add a book then a book is added successfully but your form shows an required field message 🤔 Apologies on the behalf of the previous reviewer but Kindly fix it to make your app more professional 👍